mobile/src/helpers/object/sort-by-date/sort-by-date.helper.ts

```diff
@@ -0,0 +1,26 @@
+import { compareAsc, compareDesc } from 'date-fns';
+
+type sortingType = 'desc' | 'asc';
+
+type sortItems<T> = {
+  items: T[];
+  key: keyof T;
+};
+
+const sortByDate = <T>(
+  sortItems: sortItems<T>,
+  type: sortingType = 'desc',
+): T[] => {
+  const sortingHelper = type === 'desc' ? compareDesc : compareAsc;
+  const sortingKey = sortItems.key;
+
+  if (sortingKey === 'createdAt') {
```

Comment:
We pass the sorting key to make a universal helper, and this case will only work when sortingKey equals ''createdAt'

Reply:
So remove it)